script.js

```diff
@@ -1,4 +1,67 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var SCISSORS = "scissors";
+var PAPER = "paper";
+var STONE = "stone";
+var numOfWins = 0;
+var numOfLost = 0;
+var playerName = "";
+var gameMode = "please enter player's name";
+
+var RPS = function(){
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomNumber = randomInteger + 1;
+  
+  return randomNumber;
 };
```

Comment:
good work extracting certain logic intensive code that is repeated to its own helper function